operations/kubernetes/Cert_Renewal_for_Kubernetes_and_Bare_Metal_EtcD.md

```diff
@@ -78,49 +82,60 @@ Check the expiration of the certificates.
     front-proxy-ca          Sep 02, 2030 15:21 UTC   8y              no
     ```
 
-### Backing up existing certificates
-
-1. Backup existing certificates.
+### Backup Existing Certificates
 
-    Master Nodes:
+1. Backup existing certificates on master nodes:
 
```

Comment:
If this note is made for the worker command, should one be here as well? I know that Kubernetes clusters can be configured with more than three master nodes -- not sure if we support that for CSM or not, though.

```suggestion

    **IMPORTANT:** The range of nodes below should reflect the size of the environment. This should run on every master node.

```

Reply:
We do not recommend running with more than 3 masters since that can cause Etcd latency. I will not include this change.